tests/drivers/i2c/i2c_emul/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2024 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(i2c_emul)
+
+FILE(GLOB app_sources src/*.c)
```

Comment:
`app_sources` isn't used.  Plus using GLOB in Cmake isn't recommended I believe.

Reply:
Thanks for catching